sys-devel/flang/flang-13.0.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit cmake-multilib llvm python-single-r1
```

Comment:
Please address my other comments on the old PRs, e.g llvm.org.eclass and fixing SRC_URI.